src/project/src/test/java/PersonTest.java

```diff
@@ -0,0 +1,400 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PersonTest {
+
+    private Person person;
+    private Pet pet;
+    @BeforeEach
+    void setUp() {
+        person = new Person("Mitchel", "Rose", 36, "Female");
+        pet = new Pet("gunther", 4,"Saint Bernard");
+    }
+
+    @Test
+    void getName() {
+        //Arrange
+        var person = new Person("Martin", "Red", 36, "Male");
```

Comment:
Echter hoef je ze dan hier niet meer neer te zetten en kan je direct bij de act refereren naar de person in de Setup declaratie.